hook/filter/filter_test.go

```diff
@@ -73,6 +73,7 @@ func Test_systemFileFilterHook_Hook(t *testing.T) {
 					},
 				},
 				SystemFiles: []string{
+					"/",
```

Comment:
it also requires adding application with an empty FilePath (like `helmfile` )

Reply:
Test added